SecondTask/Screens/SensorScreen.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SensorScreen.swift
+//  SecondTask
+//
+//  Created by Kalana Rathnayaka on 2021-07-14.
+//
+
+import CoreMotion
+import SwiftUI
+
+struct SensorView: View {
+    @ObservedObject var motionMagneto: MagnetoManager
+    @ObservedObject var motionAccelero: AcceleroManager
+    @ObservedObject var motionGyro: GyroManager
+
+    var body: some View {
+        NavigationView {
+            VStack {
+                Text("Magnetometer Data")
```

Comment:
Create a custom view

Reply:
https://www.hackingwithswift.com/quick-start/swiftui/how-to-create-and-compose-custom-views